Minio.Functional.Tests/FunctionalTest.cs

```diff
@@ -603,12 +601,12 @@ private static void runQuickTests(MinioClient minioClient)
                         await TearDown(minio, bucketName);
                 }
                 new MintLogger("PutObject_Test8","Tests whether PutObject with unknown stream-size passes",TestStatus.PASS,(DateTime.Now - startTime)).Log();
+            }
+            catch (Exception ex)
+            {
+                new MintLogger("PutObject_Test8","Tests whether PutObject with unknown stream-size passes",TestStatus.FAIL,(DateTime.Now - startTime),"",ex.Message, ex.ToString()).Log();
+            }
         }
-        catch (Exception ex)
```

Comment:
this exception for the outer try block is intended to catch Setup_Test failure as well as the inner using {} block. So it shouldn't be removed.

Reply:
In this case I have just changed the indentation here @poornas block is not removed..